PP_APP/server/frontend/templates/home.html

```diff
@@ -118,23 +113,24 @@ <h2 class="align-items-center text-center">&nbsp;</h2>
       </div> <!-- div class carousel -->
     </div> <!-- columns -->
   </div> <!-- row -->
-</div>
 
-<div class="container py-5" style="text-align: center">
-    <div class="row justify-content-md-center">
-          <p class="px-5 w-75" style="font-size: 14pt;"">We are using computer vision and machine learning
-            to identify packagers with the best practices.
-            Please help us  contributing pictures of your packaging so
+    <h2 class="align-items-center text-center">&nbsp;</h2>
+    <h2 class="align-items-center text-center">What are we doing about it?</h2>
+    <h6 class="align-items-center text-center">&nbsp;</h6>
```

Comment:
I know that spacing can be a real challenge so I don't want to give too hard of a time for it, but I think this is closer to the carousel than to the paragraph, which makes it look more like a caption rather than a title.

Reply:
Yeah, I definitely thought the same thing. The problem is when you look at it in the mobile view, these spaces are amplified and things appear super spaced out. It's a tough balance to strike because there are many more edge cases to deal with in terms of CSS.